src/angular/tag/tag-link.ts

```diff
@@ -0,0 +1,39 @@
+// Workaround for: https://github.com/bazelbuild/rules_nodejs/issues/1265
+/// <reference types="@angular/localize/init" />
+
+import { coerceNumberProperty, NumberInput } from '@angular/cdk/coercion';
+import { ChangeDetectionStrategy, Component, Input, ViewEncapsulation } from '@angular/core';
+
+@Component({
+  selector: 'a[sbb-tag-link]',
+  exportAs: 'sbbTagLink',
+  templateUrl: './tag-link.html',
+  styleUrls: ['./tag.css'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  encapsulation: ViewEncapsulation.None,
+  host: {
+    class: 'sbb-tag-base sbb-link-reset sbb-tag-link',
+  },
+})
+export class SbbTagLink {
+  /** Amount displayed in badge */
+  @Input()
+  get amount(): number {
+    return this._amount;
+  }
+  set amount(value: number) {
+    this._amount = coerceNumberProperty(value);
+    this._badgeDescriptionFallback =
+      typeof $localize === 'function'
+        ? $localize`:Aria label for amount of results displayed in badge pill@@sbbTagBadgePillAmountOfResults:${this.amount} results available`
```

Comment:
@kyubisation  Does this need a different i18n ID than sbb-tag ?

Reply:
If it is exactly the same text and has the same meaning, no. Otherwise, yes.

Reply:
it's exactly the same